schemas/log.yml

```
@@ -156,3 +156,50 @@
 
         According to RFCs 5424 and 3164, the priority is 8 * facility + severity.
         This number is therefore expected to contain a value between 0 and 191.
+
+    - name: syslog.version
+      level: extended
+      type: long
```

```suggestion
      type: keyword
```

Versions fields in ECS are usually `keyword` since their values are identifiers and are not expected to be used for a numeric operation or `range` query. So I'd suggest `keyword` instead here.

Also future proofs against different variations of versioning (1.1, 2.0.0-rc1, etc.)